test/api-digester/source-stability.swift.expected

```diff
@@ -7,6 +7,9 @@ Protocol IndexableBase has been removed (deprecated)
 Protocol MutableIndexable has been removed (deprecated)
 Protocol RandomAccessIndexable has been removed (deprecated)
 Protocol RangeReplaceableIndexable has been removed (deprecated)
+Func Array.append(contentsOf:) has been removed
+Func ArraySlice.append(contentsOf:) has been removed
+Func ContiguousArray.append(contentsOf:) has been removed
```

Comment:
But I guess I do need one for migration help from the `UnsafePointer` to `UnsafeBufferPointer` versions of `initialize`